todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -68,13 +89,60 @@ public static TaskDetailFragment newInstance(String taskId) {
     @Override
     public void onResume() {
         super.onResume();
-        mPresenter.subscribe();
+        bind();
     }
 
     @Override
     public void onPause() {
+        unbind();
         super.onPause();
-        mPresenter.unsubscribe();
+    }
+
+    private void bind() {
+        mSubscription = new CompositeSubscription();
```

Comment:
Yes. The main differences between MVP and MVVM:
* the ViewModel exposes streams of data to which the View can bind to. 
* the ViewModel does not contain a reference to the View.